test/extended/node/minimum_kubelet_version.go

```diff
@@ -0,0 +1,272 @@
+package node
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/onsi/ginkgo/v2"
+	g "github.com/onsi/ginkgo/v2"
+	o "github.com/onsi/gomega"
+	operatorv1client "github.com/openshift/client-go/operator/clientset/versioned"
+	"github.com/openshift/origin/pkg/test/ginkgo/result"
+	exutil "github.com/openshift/origin/test/extended/util"
+	authorizationv1 "k8s.io/api/authorization/v1"
+	v1 "k8s.io/api/core/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	clientset "k8s.io/client-go/kubernetes"
+	restclient "k8s.io/client-go/rest"
+	"k8s.io/kubernetes/test/e2e/framework"
+)
+
+const (
+	nodesGroup          = "system:nodes"
+	nodeNamePrefix      = "system:node:"
+	desiredTestDuration = 25 * time.Minute
+)
+
+var _ = g.Describe("[sig-node][OCPFeatureGate:MinimumKubeletVersion] ", func() {
+	defer g.GinkgoRecover()
+
+	oc := exutil.NewCLIWithoutNamespace("minimum-kubelet-version")
+
+	g.DescribeTable("admission", func(version string, handleErrFunc func(error)) {
+		defer updateMinimumKubeletVersion(oc, version, handleErrFunc)
+	},
+		g.Entry("should allow an empty minimum kubelet version", "", expectNoError),
+		g.Entry("should allow an old minimum kubelet version", "1.30.0", expectNoError),
```

Comment:
Maybe I don't understand something, but I think this case will set a new minimum version, which will cause a new revision. 

In that case, I think we should ensure that a parallel test (if this test runs in parallel with others) won't observe a "partial state". I think that we should make sure the servers are stable.

Reply:
good point. I have replaced those two with one other failing test. both 1.30.0 and "" will be set in the authorization test anyway, so I think this is better. now the failure cases will deny and can be run in parallel